typings/vscode-mssql.d.ts

```diff
@@ -490,120 +490,26 @@ declare module 'vscode-mssql' {
 	}
 
 	/**
-	 * Represents boolean value type deployment option.
-	 */
+	* Interface containing deployment options of boolean type
+	*/
 	export interface DacDeployOptionPropertyBoolean {
 		value: boolean;
 		description: string;
 		displayName: string;
 	}
 
 	/**
-	 * Represents object type valued deployment option.
-	 */
+	* Interface containing deployment options of integer type and hold DacFx ObjectType enum values
```

Comment:
Same as the other one - please provide a link (source preferred, docs are fine too) of the enum this is tied to